www_src/lib/api.js

```diff
@@ -62,4 +65,127 @@ module.exports = function (options, callback) {
       callback(false, body);
     }
   });
+}
+
+api.BASE_URI = 'https://webmaker-api.herokuapp.com';
+api.BASE_LOGIN_URI = 'https://id.mofostaging.net';
+
+api.AUTHENTICATE_URI = api.BASE_LOGIN_URI + '/login/oauth/access_token';
+api.SIGN_UP_URI = api.BASE_LOGIN_URI + '/create-user';
+api.USER_URI = api.BASE_LOGIN_URI + '/user';
+
+// #authenticate (object options, function callback)
```

Comment:
We are using [DocBlockr](https://packagecontrol.io/packages/DocBlockr) for JavaDoc-style documentation elsewhere.